### PR TITLE
test(core): add an integration app and symbol tracking for `@defer` feature

### PR DESCRIPTION
**Pull request by AndrewKushnir**
There are 2 commits in this PR:

- [test(core): add symbol tracking for @defer feature](https://github.com/angular/angular/commit/8118e0bac463c6a309df03845bf09733c619929e)
- [test(core): add an integration app for @defer feature](https://github.com/angular/angular/commit/ef90a9098cd34ac3b31db447c0a4638afb15efd3)

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: integration/defer/e2e/src/app.e2e-spec.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/**
+ * Note: this file contains no e2e tests, since they rely on Protractor,
+ * which requires Testability. Testability is not included by default when
+ * the `bootstrapApplication` function is used (which is the case in this app).
+ * We use this app primarily to measure payload size, so we want to keep
+ * Testability excluded.
+ */
+describe('Defer app', () => {
+  // Jasmine will throw if there are no tests.
+  it('should pass', () => {});
```
Comment (jessicajaniuk) [inline integration/defer/e2e/src/app.e2e-spec.ts:10]: Is this file necessary?

**Review by jessicajaniuk — APPROVED** [review]
LGTM

File: integration/defer/e2e/src/app.e2e-spec.ts (hunk 1 above)
Comment (josephperrott) [reply integration/defer/e2e/src/app.e2e-spec.ts:10]: Because its all generated it is necessary so that we can have one set of bazel definitions for all of the integrations.  Since it doesn't bring anything it, it will just be a constant cache hit.

**Review by josephperrott — APPROVED** [review]
LGTM

**Comment by AndrewKushnir** [comment]
**Caretaker note**: this PR is ready for merge. The only outstanding approval is `size-tracking`, but this is a newly added app, so there is no change there (LGTMs from Joey and Jess should cover that).

**Comment by atscott** [comment]
This PR was merged into the repository by commit b88cc9eccbed93e8da4e0529f3b5dc7b87da8ee4.
